src/db/mod.rs

```diff
@@ -43,6 +46,16 @@ pub trait SettingsStore {
 
     /// Returns all setting keys.
     fn setting_keys(&self) -> anyhow::Result<Vec<String>>;
+
+    /// Sets the Ethereum mapping status to "up-to-date".
+    fn set_eth_mapping_up_to_date(&self) -> anyhow::Result<()> {
```

Comment:
I'm not sure if these `eth` specific methods belong to `SettingsStore` trait. Maybe use a different trait?

Reply:
Yeah, I tend to agree. Might be confusing going forward. 
Perhaps we should introduce an actual trait like `Settings` that would have all those different methods for all sorts of settings i/o.

Reply:
Done.